sdks/python/apache_beam/io/gcp/bigtableio.py

```diff
@@ -116,11 +116,13 @@ def __setstate__(self, options):
     self.service_call_metric = None
     self.written = Metrics.counter(self.__class__, 'Written Row')
 
-  def write_mutate_metrics(self, rows):
-    for status in rows:
+  def write_mutate_metrics(self, response):
```

Comment:
If metric gathering fails for any reason, IO will still fail right?

(This is probably ok, but just checking since PR title says it will be prevented.)

Reply:
This is to prevent the cases where the callback tries to get a status code from a potential None value returned from the bigtable client.

This PR doesn't handle failures of Metrics.counter itself as it's a common module like any other Python code and it shouldn't fail for no reason.